stdlib/Markdown/src/Markdown.jl

```diff
@@ -123,7 +123,8 @@ import Base.Docs: catdoc
 catdoc(md::MD...) = MD(md...)
 
 if Base.generating_output()
-    md""  # workload to trigger precompile
+    # (representive) workload to trigger precompile; from Pkg.REPLMode.gen_help()
```

Comment:
This was better before you changed it. No need to bring in Pkg in this. It is valid for every package that parsers markdown.

Reply:
I need to have some text, to be more representative, and I think not just text, md"** text **" would do, but I'm just trying to (partially fix) regression in Pkg, along, so I chose that one... If it really matters Pkg isn't mentioned I'll change. Maybe something more is better, I just don't know the full Markdown syntax, "#", or "##" needed too, what would you suggest?